javascript-timers/timeout.js

```diff
@@ -0,0 +1,7 @@
+function timeout() {
+    setTimeout(function () {
+        var $h1 = document.querySelector("h1")
+        $h1.innerHTML = 'Hello There'
```

Comment:
Using `innerHTML` is dangerous and slow. Please use `textContent` instead.

Reply:
This has been updated.